src/uct/ib/base/ib_iface.h

```diff
@@ -195,6 +203,10 @@ struct uct_ib_iface_config {
 
     /* IB reverse SL (default: AUTO - same value as sl) */
     unsigned long           reverse_sl;
+
+    /* Time spent to prepare message request, pass it to the hardware or system
+     * software layers, and finilize it*/
```

Comment:
typo

Reply:
Fixed. Force-pushed changes because this was a fix of a typo in comment.